### PR TITLE
Added nameToHex method

### PR DESCRIPTION
**Pull request by Braunson**
As per #36

### CODE REVIEW

**Comment by Braunson** [comment]
Mind adding the `hacktoberfest` topic on the repo OR add this tag on this issue `hacktoberfest-accepted` so it counts :)

**Comment by peter279k** [comment]
If possible, could you add the tests for this `nameToHex` method?

**Comment by Braunson** [comment]
@peter279k Yep, they've been added.

File: src/Mexitek/PHPColors/Color.php [hunk 1]
```diff
@@ -232,6 +232,171 @@ public static function rgbToString(array $rgb = array()): string
...
+            'wheat' => 'F5DEB3',
+            'white' => 'FFFFFF',
+            'whitesmoke' => 'F5F5F5',
+            'yellow' => 'FFFF00',
+            'yellowgreen' => '9ACD32'
+        );
+
+        $color_name = strtolower($color_name);
+        if (isset($colors[$color_name])) {
+            return ('#' . $colors[$color_name]);
```
Comment (mikaykun) [inline src/Mexitek/PHPColors/Color.php:395]: Please remove unnecessary parentheses.
```suggestion
            return '#' . $colors[$color_name];
```

File: src/Mexitek/PHPColors/Color.php [hunk 2]
```diff
@@ -232,6 +232,171 @@ public static function rgbToString(array $rgb = array()): string
...
+        }
+
+        return ($color_name);
```
Comment (mikaykun) [inline src/Mexitek/PHPColors/Color.php:398]: Please remove unnecessary parentheses.
```suggestion
        return $color_name;
```

**Review by mikaykun — CHANGES_REQUESTED** [review]

File: tests/colorConvertHex.phpt [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    'violet' => 'EE82EE',
+    'wheat' => 'F5DEB3',
+    'white' => 'FFFFFF',
+    'whitesmoke' => 'F5F5F5',
+    'yellow' => 'FFFF00',
+    'yellowgreen' => '9ACD32'
+);
+
+foreach ($colorsToConvert as $name => $hex) {
+    $hex = '#'. $hex;
```
Comment (mikaykun) [inline tests/colorConvertHex.phpt:160]:
```suggestion
    $hex = '#' . $hex;
```

**Comment by peter279k** [comment]
After checking the latest Travis CI build log, it will present the coding style issue via `vendor/bin/phpcs --extensions=php,phpt --warning-severity=0` command:

```
$ vendor/bin/phpcs --extensions=php,phpt --warning-severity=0

FILE: /home/travis/build/mexitek/phpColors/tests/colorConvertHex.phpt

----------------------------------------------------------------------

FOUND 1 ERROR AFFECTING 1 LINE

----------------------------------------------------------------------

 160 | ERROR | [x] Expected at least 1 space before "."; 0 found

----------------------------------------------------------------------

PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY

----------------------------------------------------------------------
```

File: README.md [hunk 4]
```diff
@@ -167,6 +167,7 @@ echo $myBlue->getCssGradient(10, TRUE);
 - primozcigler
 - thedavidmeister
 - tylercd100
+- [Braunson](https://github.com/braunson)
```
Comment (peter279k) [inline README.md:170]: To be consistency, I think it should be `braunson` without link sytntax.

**Comment by peter279k** [comment]
@mikaykun, do you have any requested changes?

If not, I think we're going to be merged :).

**Review by mikaykun — APPROVED** [review]
@peter279k Looks good

**Pull request merged**